tests/flytekit/unit/core/image_spec/test_default_builder.py

```diff
@@ -34,6 +34,7 @@ def test_create_docker_context(tmp_path):
         entrypoint=["/bin/bash"],
         pip_extra_index_url=["https://extra-url.com"],
         source_copy_mode=CopyFileDetection.ALL,
+        copy=["tests/flytekit/unit/core/image_spec/test_files/file.txt", "tests/flytekit/unit/core/image_spec/test_files"],
```

Comment:
Could we programmatically generate a temporary file instead of adding a new file in the flytekit?
https://docs.python.org/3/library/tempfile.html